packages/odyssey-react-mui/src/labs/SideNav.tsx

```diff
@@ -577,16 +589,43 @@ const SideNav = ({
 }: SideNavProps) => {
   const [isSideNavCollapsed, setSideNavCollapsed] = useState(false);
   const odysseyDesignTokens = useOdysseyDesignTokens();
+  const [sideNavItemIdToScrollTo, setSideNavItemIdToScrollTo] =
+    useState<string>("");
+  const scrollToNode = useCallback<RefCallback<HTMLLIElement>>((node) => {
+    node?.scrollIntoView({
+      behavior: "instant",
+      block: "end",
+      inline: "nearest",
+    });
+  }, []);
```

Comment:
This could technically be a function at the top above the component. It's not specific to this particular component, so I would be fine having it separated out.

Another benefit of that is it could be separately tested.